src/Middlewares/CacheResponse.php

```diff
@@ -21,13 +21,15 @@ public function __construct(ResponseCache $responseCache)
         $this->responseCache = $responseCache;
     }
 
-    public function handle(Request $request, Closure $next, $lifetimeInSeconds = null): Response
+    public function handle(Request $request, Closure $next, $lifetimeInSeconds = null, ...$tags): Response
```

Comment:
Let's just only accept an array of tags.

Reply:
Can you elaborate here? My understanding is that middleware parameters can only be passed as a comma delimited string. If we only want to accept an array of tags, I think we'll have to use some other delimiter (e.g. `|`) to separate the tags in string form. My only concern then would be if tags were to contain the delimiter.